Content.Server/Physics/Controllers/RandomWalkController.cs

```diff
@@ -77,8 +77,7 @@ public void Update(EntityUid uid, RandomWalkComponent? randomWalk = null, Physic
             randomWalk.BiasVector *= 0f;
         var pushStrength = _random.NextFloat(randomWalk.MinSpeed, randomWalk.MaxSpeed);
 
-        _physics.SetLinearVelocity(uid, physics.LinearVelocity * randomWalk.AccumulatorRatio, body: physics);
-        _physics.ApplyLinearImpulse(uid, pushVec * (pushStrength * physics.Mass), body: physics);
+        _physics.SetLinearVelocity(uid, physics.LinearVelocity * randomWalk.AccumulatorRatio + pushVec * (pushStrength * physics.Mass) * physics.InvMass, body: physics);
```

Comment:
You are multiplying mass and inverse mass here. They cancel out.
```suggestion
        _physics.SetLinearVelocity(uid, physics.LinearVelocity * randomWalk.AccumulatorRatio + pushVec * pushStrength, body: physics);
```

Reply:
Edited it.